cmd/list.go

```diff
@@ -26,7 +26,7 @@ type Data struct {
 func List() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "list",
-		Short: "lists your active cloud native development environments",
+		Short: "Lists your active cloud native development environments",
```

Comment:
nit: the language we use to describe our commands is not consistent. Most CLIs use imperative mood rather than 3rd person. We should also change this for `down` and `up`.

```suggestion
		Short: "List your active cloud native development environments",
```